container-registry/container-analysis/pom.xml

```diff
@@ -0,0 +1,83 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.containerregistry</groupId>
+  <artifactId>containeranalysis</artifactId>
+  <version>1.0</version>
+  <packaging>jar</packaging>
+
+  <name>containeranalysis</name>
+  <url>http://maven.apache.org</url>
+
+  <!-- local maven repo contains container analysis client library -->
+  <repositories>
+    <repository>
+      <id>project.local</id>
+      <url>file:///${project.basedir}/repo</url>
+    </repository>
+  </repositories>
+
+  <!--
+  The parent pom defines common style checks and testing strategies for our samples.
+  Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.8</version>
+  </parent>
+
+  <properties>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <!-- [START dependencies] -->
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-containeranalysis</artifactId>
+      <version>0.1.1-0</version>
```

Comment:
0 as a suffix?  Why not Alpha, beta, eap??

Reply:
fixed